packages/block-library/src/embed/variations.js

```diff
@@ -29,13 +29,6 @@ import {
 	embedPocketCastsIcon,
 } from './icons';
 
-/** @typedef {import('@wordpress/blocks').WPBlockVariation} WPBlockVariation */
-
-/**
- * Template option choices for predefined columns layouts.
```

Comment:
Maybe we should leave the type definitions here and only update copy-pasted comment. What do you think?

Reply:
I was unsure about this, but removed it because there were no type definition comments in the variation except for the column block.

If we were to keep the type definition comment, would "Type of oEmbed provider." be appropriate?

Reply:
I also see `any` for `WPBlockVariation`, which is a bit odd since I can see the type definition here - https://github.com/WordPress/gutenberg/blob/trunk/packages/blocks/src/api/registration.js#L67-L103.

What do you think about "The embed provider services" as a comment?

This is how they are listed in the [documentation](https://wordpress.org/support/article/embeds/#list-of-sites-you-can-embed-from), and we can avoid confusion with the `Type` keyword.

Reply:
That makes sense, I have reverted to the correct comment 👍